SG Agreement.md

```diff
@@ -43,6 +43,11 @@ its interests, cast votes, and otherwise represent the interests of the [Steerin
 [Steering Group Member] may replace its [Steering Group Representative] at any time in its sole
 discretion.
 
+Each [Steering Group Member] may optionally appoint an individual
+"<a id="alternate-steering-group-representative">**Alternate Steering Group Representative**</a>". The
+[Steering Group Representative] may delegate any of their rights and responsibilities to the
```

Comment:
I think we probably need a little bit more here.  Is the expectation this is temporary?  That we'll have multiple participants in our SG meetings, and it's only when it comes to voting that we ensure it's one Member one vote?  Is it a single additional individual, or multiple?

Reply:
IMO, doesn't need to be temporary. Maybe allow both to participate in SG meetings (though I expect typically only one person will be present at a time). A single additional individual seems sufficient.

Reply:
Is the current text ok? The voting is aspect I think boils down to clarifying the word "party" in https://whatwg.org/sg-agreement#qualified-voting-party (if neccessary).